e2e-tests/commands/insurance/complete-and-submit-broker-details-form.js

```diff
@@ -1,9 +1,12 @@
 /**
  * completeAndSubmitBrokerDetailsForm
  * Complete and submit "broker details" form
+ * @param {String} name: Broker name
+ * @param {String} email: Broker email
+ * @param {String} fullAddress: Broker's full address
  */
-const completeAndSubmitBrokerDetailsForm = () => {
-  cy.completeBrokerDetailsForm();
+const completeAndSubmitBrokerDetailsForm = ({ name, email, fullAddress }) => {
```

Comment:
do we need default values here as well?

Reply:
No, it'll always go to `completeBrokerDetailsForm` which has default values. Having them here would just be duplicated. We have this pattern in several places.